### ISSUE TITLE
Ignore wrong commands

### ISSUE THREAD

**Issue by ghost** [dmitmel/cmp-cmdline-history#4]
Hi, I used this plugin for over 2 months and I love it alot,
I think it can be even better if it can ignore wrong commands or I can manually delete command history
Is there any way to do this?
Thanks

**Comment by dmitmel** [comment]
Vimscript commands are very hard to parse, I am unsure if this can be done performantly. However,

> or I can manually delete command history

You can use the [`histdel`](https://neovim.io/doc/user/builtin.html#histdel()) function for that.

**Comment by ghost** [comment]
thank you

**Comment by daephx** [comment]
Necro posting as I don't think a new issue is warranted. But for my own curiosity I wanted to ask, is it not possible or insufficient to detect whether a user inputed command responded with an error and ignore it from being saved in history?
